### PR TITLE
Add product and put product

### LINKED ISSUE #45: 新增商品 修改商品

### PR DESCRIPTION
**Pull request by wj461**

### CODE REVIEW

**Review by nonorze-VOXX — DISMISSED** [review]
LGTM

File: backend/product/delivery/product_handler.go [hunk 1]
```diff
@@ -35,3 +38,58 @@ func (s *ProductHandler) GetProductById(c *gin.Context) {
 	}
 	c.JSON(200, products)
 }
+
+func (s *ProductHandler) PostProductByStoreId(c *gin.Context) {
```
Comment (huhuhu0420) [inline backend/product/delivery/product_handler.go]: rename the function that  [human](https://en.wikipedia.org/wiki/Human) can understand
e.g. AddProduct

Comment (wj461) [reply backend/product/delivery/product_handler.go]: fix

**Review by nonorze-VOXX — APPROVED** [review]
Can run for me

**Pull request merged**